server.js

```diff
@@ -1,23 +1,110 @@
 const express = require("express");
-const cors = require("cors");
+
 
 const app = express();
+const bodyParser = require("body-parser");
+app.use(bodyParser.urlencoded({
+    extended: true
+}));
+
+app.use(bodyParser.json())
+app.use(express.json())
+
```

Comment:
can You explain to me why you used .json() on the Bodyparser and express Sulaiman.